MinecraftClient/Protocol/Handlers/Protocol18.cs

```diff
@@ -1489,5 +1567,22 @@ public bool SendCloseWindow(int windowId)
             catch (System.IO.IOException) { return false; }
             catch (ObjectDisposedException) { return false; }
         }
+        public bool SendUpdateSign(Location sign, string line1, string line2, string line3, string line4)

```

Comment:
Is there a limit on line length? Maybe enforce the limit here to avoid crash/disconnect on lines too long?